com.playeveryware.eos/Documentation~/full_guide/01_starthere.md

```diff
@@ -0,0 +1,126 @@
+Each Client is assigned to one Client Policy.
+Client Policies are configurable sets of rules for what permissions a Client can perform. For example, a Client Policy determines the EOS SDK's ability to look up information about a player's friend list.
+
+First, create a Client policy. Press the `Add new policy` button to bring up the dialogue.
+The Client Policy needs a name, as well as select the details of a policy.
+If you select a preset, the permissions for that policy preset will be displayed in the dialogue.
+Custom policies can be defined here, and changed at any time. You don't need to commit to the final details at this time.
+The most permissive default policy is `GameClient /w UnlockAchievements`, which will give the Client Policy all of the permissions necessary to perform most actions.
+
+When the Client Policy is created, next create a client. Press `Add new client` to bring the Add new client dialogue.
+This will require a name and a Client Policy. Use the previously created Client Policy.
+`Trusted Server IP allow list` and `Epic Account Service Redirect URL` are not required to be filled at this time.
+
+# Creating an Application
+
+Navigate to the `Epic Account Services` page. The landing will show you all configured Applications, which is empty at first.
+See [Epic's documentation on Applications in the Developer Portal Introduction](https://dev.epicgames.com/docs/dev-portal/dev-portal-intro#application) for more information on Applications.
+An Application is collection of information about your game, which is later used to provide the Epic Game Store with essential information and permissions.
+Use `Create Application` to get started. Immediately when you press this button, an Application will be created for you.
+This will bring up a dialogue with three major steps; `Brand Settings`, `Permissions`, and `Linked Clients`.
+
+`Brand Settings` contains information about your game such as the Application's website.
+To utilize the EOS Plugin for Unity, you do not need to fill this page out at this time.
+
+`Permissions` will set the essential information that each user needs to consent to in order to use your application.
+This can be adjusted later. For full use of the EOS Plugin for Unity, consider marking `Online Presence` and `Friends` as Required.
+
+`Linked Clients` matches an Application to the Client that dictates its policies and further permissions.
+Assign the Client Policy created earlier to this.
+
+When you exit this dialogue, the created Application will appear in the Applications page.
+Navigating back to the `Product Settings` page, `Sandboxes` and `Deployments` will automatically be made for Dev, Stage, and Live.
+
+# Add the EOS Plugin for Unity to Your Game
+
+With a Client Policy, Client, and Application created, you're ready to start adding the EOS Plugin for Unity to your game.
+Follow [the Add Plugin documentation](/com.playeveryware.eos/Documentation~/add_plugin.md) to add the Plugin to the project.
```

Comment:
link text should be, "our documentation for adding the plugin to your project". (and the remainder of the sentence should be omitted since the link text accomplishes the same thing).

Reply:
Implemented as suggested.